cmd/state-handler/main.go

```diff
@@ -54,7 +54,10 @@ func main() {
 			*hostname = envHostname
 		}
 	}
-
+	if err := nmstatectl.Check(); err != nil {
+		klog.Fatalf("Error checking nmstatectl: %v\n", err)
```

Comment:
Please provide a nicer log message, telling we failed to execute nmstatectl, please check that NetworkManager is running on node ...

Reply:
When a user sees the log, they should be able to resolve the problem without reading/searching the traceback.

Reply:
@phoracek Can we check from here processes running at k8s hosts ?

Reply:
@qinqon no, the error message should be hint/suggestion. We don't need to know for sure, just give user the most likely cause.

Reply:
Ok, removed go traceback but keeped nmstatectl python error and added possible check on nodes.